enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+
+As OLM is transitioning to declarative config, which is a more human readable and consumable format of a catalog that is in the form of `JSON/YAML`, Operator SDK should transition the existing `run bundle(-upgrade)` commands to support declarative config index images and default to using File-Based Catalogs so that Operator authors can continue using Operator SDK to install/upgrade their operator bundles using the new FBC format.
+
+This proposal discusses changes to the `run bundle` functionality to be able to support File-Based Catalogs and how the registry pod should be created to support the generated FBC format i.e., to modify/replace the existing opm commands used to create the ephemeral index image during registry pod creation by leveraging the Operator Registry APIs.
+
+### Goals
+
+- Operator SDK’s `run bundle` command should work with File-Based Catalogs and be able to install an operator bundle using a valid FBC.
+
+- Operator SDK should insert a bundle image into an existing index image that gets passed in as a parameter in the `run bundle` command.
+
+- Operator SDK's `run bundle` command should still be able support SQLite index images along with the FBC images until `opm registry add` and `opm index` and other SQLite-related commands are removed in a future OpenShift release.
+
+- The CLI for `run bundle` command should remain exactly the same i.e. no user-facing changes.
+
+### Non-Goals
+
+- Operator SDK’s `run bundle` and `run bundle-upgrade` commands share a lot of common functionality and this EP does not discuss the impact on the run bundle-upgrade command. A separate EP is required for the `run bundle-upgrade` command.
+
+- Migrating from old catalog images to newer catalog versions with FBC support is not a goal of this EP.
+
+- Another non-goal, is that `run bundle` is **NOT** intended to help users build and manage File-Based indexes. It is an opinionated command to help users test simple bundle-based execution scenarios with ephemeral indexes.
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+As an Operator SDK user, I want to be able to install an operator bundle with File-Based Catalog support using OLM without having to provide a catalog.
+
+#### Story 2
+
+As an Operator SDK user, I want to be able to install an operator bundle with File-Based Catalog support using OLM when I want to use an existing File-Based Catalog as the index image.
```

Comment:
```suggestion
As an Operator SDK user, I would like to be able to inform an index using the FBC format via the `--index` option provided on the command `run bundle` so that, I can use this option with the index catalog built from OperatorHub (quay.io/operatorhubio/catalog) for example. 
```